web/yo/app/views/realtimeString.html

```diff
@@ -51,7 +51,7 @@
                             chosen allow-single-deselect="true"
                             ng-model="data[key]"
                             ng-options="opt.value as opt.label for opt in o">
-                        <option value="NA"></option>
```

Comment:
Explanation here?

Reply:
This is a bug fix since I run into issue https://github.com/harvesthq/chosen/issues/1942.